Source/Scene/Camera.js

```diff
@@ -2738,6 +2738,7 @@ define([
     Camera.clone = function(camera, result) {
         if (!defined(result)) {
             result = new Camera(camera._scene);
+            result.frustum = camera.frustum.clone();
```

Comment:
Why is this clone needed when there is a clone below?  Is this for allocating the right type of frustum?  If so, we should add a comment here since it is non-obvious.